config/jobs/kubernetes/sig-k8s-infra/trusted/releng/releng-trusted.yaml

```diff
@@ -47,13 +47,22 @@ postsubmits:
         - cip
         - --thin-manifest-dir=/home/prow/go/src/github.com/kubernetes/k8s.io/k8s.gcr.io
         - --confirm
+        # TODO: these are *not* threads 🤷‍♂️
+        # https://www.geeksforgeeks.org/golang-goroutine-vs-thread/
+        #
+        # this should probably be a multiple of cores below
+        - --threads=14
+        env:
+        - name: GOMAXPROCS
+          value: "7"
         resources:
+          # request most of one node 🚀
           requests:
-            cpu: 2
-            memory: "4Gi"
+            cpu: 7
+            memory: "40Gi"
```

Comment:
we're unlikely to need this much ram, but the nodes have 50gb, and 7/8 core jobs will prevent any other reasonable job with at least one core req from being scheduled anyhow